src/hooks/useGlobalAppState.ts

```diff
@@ -57,8 +57,6 @@ const onStateChange = (
  */
 
 export const useGlobalAppState = (): void => {
-  // Ref to track if the logic has already been executed
-  const hasExecuted = useRef(false)
```

Comment:
I agree in principle, we must make sure that if for whatever reason this hook is launched twice, it doesn't break the app at other places though

Reply:
Yes, but it was not used anymore. If you check the current code, there is no place where we call `hasExecuted.current = true` anymore.

This has been remove in https://github.com/cozy/cozy-flagship-app/commit/dbfcf54caff9511d3f3cefa1ed015f60f8c6b023#diff-cee1a6f779963233af0e8891de5e8aec2ca24cdb7e9ba8cae9b8ec4f908dc51d

Also the entire block that used it has been removed. It was just about checking if `getIsSecurityFlowPassed()` was `true` or `false` to display a dedicated log in the console